messenger.rst

```diff
@@ -658,7 +658,7 @@ Graceful Shutdown
 
 If you install the `PCNTL`_ PHP extension in your project, workers will handle
 the ``SIGTERM`` POSIX signal to finish processing their current message before
-exiting.
+terminating.
```

Comment:
I am not sure about this, is there a technical difference between exiting and terminating?

cc @stof @nicolas-grekas

Reply:
The word terminating is more precise in meaning. Exiting from what and to where, and what happens after it exited? I know tech people use the term often, but in formal writing, such as documentation, it's probably better to use terminate.